tools/travis_mvn_watchdog.sh

```diff
@@ -225,9 +226,14 @@ echo "MVN exited with EXIT CODE: ${EXIT_CODE}."
 rm $MVN_PID
 rm $MVN_EXIT
 
-check_shaded_artifacts
-
-put_yarn_logs_to_artifacts
```

Comment:
This is removed here, is that on purpose?
Are the yarn logs now handled differently?